src/content/advanced/ingress/configuration/index.md

```diff
@@ -421,57 +395,14 @@ data:
       log-format-upstream: "MY EDITED LOG FORMAT - $status $body_bytes_sent $http_referer"
 ```
 
-Any defaults that we override are visible in the following `values.yaml` file, under the `configmap` key. [Check this values.yaml file in v1.6.10](https://github.com/giantswarm/nginx-ingress-controller-app/blob/v1.6.10/helm/nginx-ingress-controller-app/values.yaml) as an example.
-
-Do make sure you look at the right tag of that repository, when reading this file check that the tag
-corresponds to the version of the nginx-ingress-controller-app running on your cluster.
+However keep in mind that with great power comes great responsibility.
```

Comment:
I kept this since this is ingress controller business :) but also fine to drop if it's overkill.